api/envoy/data/accesslog/v2/accesslog.proto

```diff
@@ -224,6 +225,19 @@ message TLSProperties {
 
   // SNI hostname from handshake.
   string tls_sni_hostname = 3;
+
+  message CertificateProperties {
+    // The URI in the SAN field of the certificate.
+    string uri_san = 1;
```

Comment:
this is not quite reflect what is in X.509, perhaps something like:
```
message SubjectAltName {
  oneof {
    string uri;
    string dns;
  }
}
repeated SubjectAltName subject_alt_names;
```

@PiotrSikora?

Reply:
Yeah, seems like SAN should be a repeated.

Reply:
Sorry. I did not see this earlier. We already have `CertificateDetails` message which does what  @lizan  mentioned above. Probably there is a chance to reuse here https://github.com/envoyproxy/envoy/blob/4a5f85809cef13be47873483789a5a7bb64ded7c/api/envoy/admin/v2alpha/certs.proto#L30